frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -24,6 +26,9 @@ class LibraryPage extends React.Component {
             ]}
           />
         </Grid>
+        <Grid item style={{ flexGrow: 1 }}>
+          <EmptyLibrarySubPage />
```

Comment:
Not sure about this naming, so feel free to suggest anything

Reply:
EmptyLibraryContent? Maybe something like that

Reply:
Good one 👍🏻